setup.py

```diff
@@ -1,13 +1,21 @@
-import setuptools
+from setuptools import find_packages, setup
```

Comment:
Is there any particular reason to pull these into our namespace?

Reply:
well, i was going through the ```setup.py``` file of some open source repositerioes and i found that a lot of the big names in software uses ```from ... import ...``` instead of ```import``` so i thought it would be nice to add it here.

But to be true when i digged more into this i found no real gain for this solution, so if you don't prefer it i have no problem taking it down.